SI/Sem.02/Solutions/Task05.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+	double sideA, sideB, sideC;
+	cin >> sideA >> sideB >> sideC;
+	if ((sideA + sideB) > sideC && (sideA + sideC) > sideB && (sideB + sideC) > sideA)
```

Comment:
if ((a + b > c) && (a + c > b) && (b + c > a)),
would it be better to group the cases, rather than the addition?